qa/tasks/deepsea.py

```diff
@@ -141,7 +148,7 @@ def _install_deepsea(self):
         deepsea_ctx['deepsea_installed'] = True
 
     def _install_deepsea_from_source(self):
-        log.info("WWWW: installing deepsea from source")
+        log.info(self.log_anchor + ": installing deepsea from source")
```

Comment:
What if I wish set log_anchor, to None or empty string?

Reply:
updated; please have another look